grails-app/domain/org/broadinstitute/orsp/Issue.groovy

```diff
@@ -60,39 +66,116 @@ class Issue {
+            it.name == IssueExtraProperty.ACCURATE
+        }?.value?.toString())
+    }
+
+    transient String getAffiliationOther() {
+        getExtraProperties().find { it.name == IssueExtraProperty.AFFILIATION_OTHER }?.value
+    }
+
+    transient String getDbgap() { getExtraProperties().find { it.name == IssueExtraProperty.DBGAP }?.value?.toString() }
+
+    transient String getCodes() { getExtraProperties().find { it.name == IssueExtraProperty.CODES }?.value }
+
+    transient String getIrb() { getExtraProperties().find { it.name == IssueExtraProperty.IRB }?.value }
+
+    transient String getRationale() { getExtraProperties().find { it.name == IssueExtraProperty.RATIONALE }?.value }
+
+    transient String getResearch() { getExtraProperties().find { it.name == IssueExtraProperty.RESEARCH }?.value }
+
+    transient String getResponsible() { getExtraProperties().find { it.name == IssueExtraProperty.RESPONSIBLE }?.value }
+
+    transient String getSource() { getExtraProperties().find { it.name == IssueExtraProperty.SOURCE }?.value }
+
+    transient String getIdentifiable() {
+        getExtraProperties().find { it.name == IssueExtraProperty.IDENTIFIABLE }?.value
+    }
+
+    transient String getDeceased() { getExtraProperties().find { it.name == IssueExtraProperty.DECEASED }?.value }
+
+    transient String getSubmissionType() {
+        getExtraProperties().find { it.name == IssueExtraProperty.SUBMISSION_TYPE }?.value
+    }
+
+    transient String getFeeForService() {
+        getExtraProperties().find { it.name == IssueExtraProperty.FEE_FOR_SERVICE }?.value
+    }
+
+    transient String getCollHasIdentity() {
+        getExtraProperties().find { it.name == IssueExtraProperty.COLL_HAS_IDENTITY }?.value
+    }
+
+    transient String getReviewCategory() {
+        getExtraProperties().find { it.name == IssueExtraProperty.REVIEW_CATEGORY }?.value
+    }
+
+    transient String getCommerciallyAvailable() {
+        getExtraProperties().find { it.name == IssueExtraProperty.COMMERCIALLY_AVAILABLE }?.value
+    }
+
+    transient String getInteract() { getExtraProperties().find { it.name == IssueExtraProperty.INTERACT }?.value }
+
+    transient Collection<String> getActorUsernames() {
+        getExtraProperties().findAll { it.name == IssueExtraProperty.ACTOR }.collect { it.value }
+    }
+
+    transient Collection<String> getPIs() {
+        getExtraProperties().findAll { it.name == IssueExtraProperty.PI }.collect { it.value }
+    }
+
+    transient Collection<String> getPMs() {
+        getExtraProperties().findAll { it.name == IssueExtraProperty.PM }.collect { it.value }
+    }
+
+    transient Collection<String> getAffiliations() {
+        getExtraProperties().findAll { it.name == IssueExtraProperty.AFFILIATIONS }.collect { it.value }
+    }
+
+    transient Collection<String> getNotResearch() {
+        getExtraProperties().findAll { it.name == IssueExtraProperty.NOT_RESEARCH }.collect { it.value }
+    }
```

Comment:
I would prefer to revert this formatting change. For large, repetitive blocks like this, I like the condensed format better.